small-talk/util/socket.js

```diff
@@ -0,0 +1,13 @@
+import { io } from "socket.io-client";
+
+// Connect to the server
+const URL = "http://localhost:8000";
+// set to false for now, can be set to true for auto connect later
+const socket = io(URL, { autoConnect: false });
+
+// catching all the listeners
+socket.onAny((event, ...args) => {
+  console.log(event, args);
+});
+
+export default socket;
```

Comment:
Very nice way to create a separated socket creation that's testable and reusable

Reply:
Could possibly rename to getDefaultSocket or something to indicate its use